controllers/awsadapterconfig_controller.go

```diff
@@ -58,9 +57,13 @@ func (r *AWSAdapterConfigReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	l := log.FromContext(ctx)
 
 	objOld := &securityv1alpha1.AWSAdapterConfig{}
-	err := r.Get(ctx, types.NamespacedName{Name: req.Name, Namespace: req.Namespace}, objOld)
+	err := r.Get(ctx, req.NamespacedName, objOld)
 	if err != nil {
-		l.Error(err, "error occurred while retrieving awsadapterconfig")
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	if !objOld.DeletionTimestamp.IsZero() {
+		l.Info("Deleting AWSAdapterConfig")
```

Comment:
Slight modification to the message Abhinav. "Warning: AWSAdapterConfig is being deleted. Hope this is done intentionally."

Reply:
I've removed `AWSAdapterConfig` from this message since every log in our case contains the info about the concerned resource, such as below:

```bash
1.6711259854429429e+09  INFO    WARNING: Resource deleted or does not exist     {"controller": "awsadapterconfig", "controllerGroup": "security.nirmata.io", "controllerKind": "AWSAdapterConfig", "AWSAdapterConfig": {"name":"test-config","namespace":"default"}, "namespace": "default", "name": "test-config", "reconcileID": "d61d531d-2be5-4113-9e40-db91aa59fa52"}
```